com.googleplaymusicdesktopplayer.GPMDP.json

```diff
@@ -37,8 +35,8 @@
         "ar x gpmdp.deb",
         "tar -xf data.tar.xz",
         "cp -r usr/* /app",
-        "mkdir -p /app/share/icons/hicolor/512x512/apps",
-        "mv /app/share/pixmaps/google-play-music-desktop-player.png /app/share/icons/hicolor/512x512/apps/",
```

Comment:
This was done for a reason.

Reply:
I changed that line to create a 1024x1024 folder because the icon in the package has changed and is too high resolution for 512x512 (which was why the first build failed).